docs/deployment/image_vector.md

```diff
@@ -40,18 +41,19 @@ images:
   tag: "3.5"
   architectures:
   - amd64
+  - arm64
 - name: pause-container
   sourceRepository: github.com/kubernetes/kubernetes/blob/master/build/pause/Dockerfile
   ref: registry.k8s.io/pause:3.5
   architectures:
+  - amd64
   - arm64
 - name: pause-container
   sourceRepository: github.com/kubernetes/kubernetes/blob/master/build/pause/Dockerfile
   ref: registry.k8s.io/pause:3.5
   architectures:
   - amd64
   - arm64
```

Comment:
I think this part was fine with just having one architecture per container image. Here we want to highlight that you can specify different things per architecture. Now, we have ambiguous results instead.